### PR TITLE
update us sen yamls; create AR lt gov yaml; delete WI yaml; edit DE yaml

### PR DESCRIPTION
**Pull request by OdeclasV**
- Adding hard wait to FCAS22198_248511 
- deleting `max_length` options from US Sens yaml. I checked each one and the max lengths are no longer needed
- delete WI yaml
- creating Lt Gov of AR yaml 
Edit:
- updating a DE senator yaml

### CODE REVIEW

File: members/SARL35_283916.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+bioguide: SARL35_283916
```
Comment (tirish) [inline members/SARL35_283916.yaml:1]: Do you want to keep with the name comments at the top? I don't really care, just checking with you

File: members/SARL35_283916.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+contact_form:
+  method: POST
```
Comment (tirish) [inline members/SARL35_283916.yaml:3]: lets go ahead and start leaving this out of new yamls since it is unused

**Review by tirish — APPROVED** [review]

File: members/SARL35_283916.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/SARL35_283916.yaml:2]: yeah, why not. Doesn't hurt

File: members/SARL35_283916.yaml (hunk 2 above)
Comment (OdeclasV) [reply members/SARL35_283916.yaml]: deleted

**Pull request merged**